Education_Pathways/frontend/src/api.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: "https://lab3-docker.herokuapp.com/"
```

Comment:
If we make this our base URL, then we won't be able to see our changes to the backend (unless we push to Heroku). To view changes in the backend, make sure to use baseURL: "http://localhost:5000/"